src/app/router/Prompt/Prompt.tsx

```diff
@@ -46,6 +46,18 @@ const routeParams: {
 } = { origin, args, action };
 
 function Prompt() {
+  routeParams.action = "confirmPayment";
+  //routeParams.args.lnurlDetails = {}
+  routeParams.args = {
+    amountEditable: true,
+    memoEditable: true,
+    invoiceAttributes: {
+      destination: "test string",
+      customRecords: {},
+      valueSat: "string",
+    },
+    origin: {},
+  };
```

Comment:
be very very very careful with such test data hacks. and actually never use those. if you need something like that then we need to look at the flow to make it easier that in development this actually can be tested. 

using the app in a real world context is most important also during development.